test/user_defined_output/fake_plugin/fake_user_defined_output.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "envoy/registry/registry.h"
+
+#include "nighthawk/user_defined_output/user_defined_output_plugin.h"
+
+#include "external/envoy/source/common/common/statusor.h"
+
+#include "api/client/options.pb.h"
+#include "api/client/service.grpc.pb.h"
+
+#include "test/user_defined_output/fake_plugin/fake_user_defined_output.pb.h"
+
+namespace Nighthawk {
+
+/**
+ * UserDefinedOutputPlugin for testing: ...
```

Comment:
Can we expand the `...` and finish the comment?

Ideally we should describe the behaviors of the fake plugin, either here or on the individual methods. The comment here could focus on how the plugin should be used.